assets/other-scripts/emails/style.scss

```diff
@@ -1,3 +1,8 @@
 .editor-post-title {
 	display: none;
 }
+
+.editor-styles-wrapper .block-editor-block-list__layout .wp-block-button {
+	margin-left: 0 !important;
+	margin-right: 0 !important;
+}
```

Comment:
Added these styles to resolve an issue where the button block was forced to be centered in the editor. This overrides the behavior in the editor, but actual emails will still see centered buttons for some reason. (Aside from our updated OTP default template)